barretenberg/cpp/src/barretenberg/sumcheck/sumcheck_round.hpp

```diff
@@ -103,15 +104,15 @@ template <typename Flavor> class SumcheckProverRound {
     barretenberg::Univariate<FF, BATCHED_RELATION_PARTIAL_LENGTH> compute_univariate(
         ProverPolynomialsOrPartiallyEvaluatedMultivariates& polynomials,
         const proof_system::RelationParameters<FF>& relation_parameters,
-        const barretenberg::PowUnivariate<FF>& pow_univariate,
-        const FF alpha)
+        const barretenberg::PowPolynomial<FF>& pow_polynomial,
+        const AlphaType alpha)
     {
-        // Precompute the vector of required powers of zeta
-        // TODO(luke): Parallelize this
+        // Compute the constant contribution of pow polynomials for each edge as the product of c_l and the elements of
```

Comment:
Can you add more to this comment? Seems like it would be helpful but I'm still confused e.g. c_l is not very explicitly defined anywhere.

Reply:
c_l (the notation was already there) is the partial_evaluation_result in the pow polynomial. I renamed it from `partial_evaluation_constant` to `partial_evaluation_result` since it's not a constant, it gets updated at every round. Updated the comment